### PR TITLE
Arcmap

### PR DESCRIPTION
**Pull request by jterstriep**
Created new tutorial on accessing BetyDB from ArcMap.

### CODE REVIEW

**Comment by dlebauer** [comment]
One question: is this a process that we can open up to the public or should we add additional instructions for setting up a mirror using load.bety.sh (see https://github.com/terraref/reference-data/issues/111#issuecomment-292604008)?

**Comment by jterstriep** [comment]
I believe the current approach is appropriate for a small number of closely associated researchers who would benefit from direct access. It's not the most user-friendly of mechanisms. But others should weigh in on this.
Using load.bety.sh to create a local system pre-supposes running a postgres server somewhere. That's not impossible but installing and administering postgres might be beyond the scope of what researcher want to undertake. We could focus on providing docker images but that has its own infrastructure requirements for the researcher.

File: accessing-betydb-with-arcmap.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Accessing BetyDB from ArcMap
+Interested researchers can access BetyDB directly from ArcMap. In some
```
Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: BetyDB --> BETYdb

File: accessing-betydb-with-arcmap.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+cases direct access can simplify manipulation of the BetyDB data but
+this convenience must be weighed against a more complex setup, limits of
+ArcMap compatibility, and additional complexity of extracting data from
+a SQL database.
+
```
Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: Mention that - these instructions are intended for collaborators within the TERRA program. For users outside the TERRA program, access directly to the Postgres database requires setting up a clone. Instructions for using load.bety.sh should be copied from here: https://github.com/terraref/reference-data/issues/111

File: accessing-betydb-with-arcmap.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Accessing the production BetyDB requires creating a secure shell tunnel
+(SSH) to a remote server. After creating the tunnel the database is accessed
+as though it was available on the local machine. A step-by-step process
+is given below.
+
+### Request Access
+Start by [requesting access to the
+BetyDB](https://identity.ncsa.illinois.edu/join/TU49BUUEDM)
```
Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: Not sure if we should make this link public (@robkooper?)

File: accessing-betydb-with-arcmap.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+(https://identity.ncsa.illinois.edu/join/TU49BUUEDM) by following the
+link. This will take you to the NCSA identity service. If you do not have
+an NCSA account, you will be asked to create one. This account and
+password will be used to login to the database server. 
+
+Access will generally be granted within 24-hours.
```
Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: is (currently) only available to the TERRA REF team.

Comment (jterstriep) [inline accessing-betydb-with-arcmap.md]: This approach is supported by NCSA and designed to be public. Using the link will not allow users to self-register. They are only requesting access that will be approved manually by Rob.

File: accessing-betydb-with-arcmap.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+After confirming access to bety6 logout by typing `exit`.
+
+### Create SSH Tunnel to BETYdb 
+
+The following command will create an `SSH tunnel` from your computer to the BETYdb server: 
+
+```sh
+ssh -L 5432:localhost:5432 <login>@bety6.ncsa.illinois.edu
+```
```
Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: @jterstriep when I try this I get the error

>  ssh -L 5432:localhost:5432 dlebauer@bety6.ncsa.illinois.edu
bind: Operation not permitted
channel_setup_fwd_listener_tcpip: cannot listen to port: 5432
Could not request local forwarding.

Note that I am using MobaXTerm - which is built on PuTTY. I couldn't launch a terminal from the PuTTY software.

Comment (dlebauer) [inline accessing-betydb-with-arcmap.md]: I had postgres installed on Windows. I am uninstalling and will re-try

**Pull request merged**